applications/RANSApplication/custom_utilities/test_utilities.cpp

```diff
@@ -116,11 +116,11 @@ ModelPart& CreateScalarVariableTestModelPart(
         BufferSize);
 
     if (DoInitializeElements) {
-        r_model_part.Elements().front().Initialize();
+        r_model_part.Elements().front().Initialize(r_model_part.GetProcessInfo());
```

Comment:
Shouldn't we create a `const` reference in here?

Reply:
not needed, the only version is with const

Reply:
the deprecated version is without argument, so it cannot get confused